packages/ipfs/README.md

```diff
@@ -942,6 +942,29 @@ const node = await IPFS.create({
 
 The code above assumes you are running a local `signaling server` on port `9090`. Provide the correct values accordingly.
 
+**Note:** We host a signalling server at star-signal.cloud.ipfs.team that can be used for practical demos and experimentation, it should not be used for apps in production. A libp2p-webrtc-star address, using the signalling server we provide, looks like:
+
+/dns4/star-signal.cloud.ipfs.team/wss/p2p-webrtc-star/p2p/<your-peer-id>
```

Comment:
This address is going to end up changing when we update the deploy, which is in progress. This server is currently not stable and is old.

Reply:
Yeah, I will add this once we have the new server deployed